### ISSUE TITLE
Add objects_to_include parameter to unnest_job_summary_data_json_object()

### ISSUE THREAD

**Issue by pbchase** [ctsit/redcapcustodian#112]
Add `objects_to_include` parameter to `unnest_job_summary_data_json_object()` (See PR #111). Make sure to document the new parameter and update the test.

**Comment by ChemiKyle** [comment]
`devtools::check()`

```R
  --- re-building ‘job_logging.Rmd’ using rmarkdown
  Quitting from lines 18-22 (job_logging.Rmd) 
  Error: processing vignette 'job_logging.Rmd' failed with diagnostics:
  dot-env file does not exist
  --- failed re-building ‘job_logging.Rmd’
```

**Comment by pbchase** [comment]
Addressed by PR #115
